go/plugins/dotprompt/genkit.go

```diff
@@ -137,8 +140,16 @@ func (p *Prompt) Register() error {
 	if p.Variant != "" {
 		name += "." + p.Variant
 	}
-
-	p.action = ai.DefinePrompt("dotprompt", name, nil, p.buildRequest, p.Config.InputSchema)
+	// TODO: Unwind this change once Monaco Editor supports newer than JSON schema draft-07.
+	p.InputSchema.Version = ""
+	p.action = ai.DefinePrompt("dotprompt", name, map[string]any{
```

Comment:
I think the code will be easier to read if written as

```Go
    metadata := map[string]any{
        "prompt": ...
        ...
    }
    p.action = ai.DefinePrompt("dotprompt", name, metadata, p.buildRequest, p.Config.InputSchema)
```

Reply:
Done.